docs/documentation/functions/Install-SddcCertificate.md

```diff
@@ -0,0 +1,87 @@
+# Install-SddcCertificate
+
+## SYNOPSIS
+Connect to SDDC Manager to install/replace the signed certificates for all components associated with the given workload domain.
+
+## SYNTAX
+
+```
+Install-SddcCertificate [-server] <String> [-user] <String> [-pass] <String> [-workloadDomain] <String>
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Install-SddcCertificate will request SDDC Manager to connect to Certificate Authority to sign the generated Certificate Signing Request files for all components associated with the given workload domain.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Install-SddcCertificate -server sfo-vcf01.sfo.rainpole.io -user administrator@vsphere.local -pass VMw@re1! -workloadDomain sfo-w01
+This example will connect to Sddc Manager to install/replace the signed certificates for a given workload domain.
```

Comment:
```suggestion
This example will connect to SDDC Manager to install/replace the signed certificates for a given workload domain.
```

Reply:
addressed in (c468fc14c2f684881d0da7753d1082bc7e9e70fc)